packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -1270,7 +1272,8 @@ const TESTS: [string, any, Record<string, any>?][] = [
     ])],
     ['html', [
       { test: 'include', expected: 'Recovery phone removed' },
-      { test: 'include', expected: 'Your recovery phone has been removed from your two-step authentication settings. You still have your backup authentication codes available for use.' },
+      { test: 'include', expected: 'Your recovery phone has been removed from your two-step authentication settings.' },
+      { test: 'include', expected: 'You can still use your backup authentication codes to sign in if you aren\'t able to use your authenticator app.' },
```

Comment:
```suggestion
      { test: 'include', expected: 'You can still use your backup authentication codes to sign in if you arenʼt able to use your authenticator app.' },
```

Reply:
TIL my editor auto tranforms this when pasted.